sdk/search/Azure.Search.Documents/samples/Sample07_VectorSearch.md

```diff
@@ -18,7 +18,7 @@ We will create an instace of `SearchIndex` and define `Hotel` fields.
 string vectorSearchConfigName = "my-vector-config";
 int modelDimensions = 1536;
 
-string indexName = "Hotel";
+string indexName = "hotel";
```

Comment:
This is just a snippet. We need to update the index name both [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/tests/Samples/Sample07_VectorSearch.cs#L320) and [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/Azure.Search.Documents/tests/Samples/Sample07_VectorSearch.cs#L378) in the codebase. Afterward, execute the [snippet generator](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets) using this command:
```ps1
eng\scripts\Update-Snippets.ps1 search
```
Running this script will refresh all the snippets in all the markdown files.